fish_prompt.fish

```diff
@@ -407,6 +406,17 @@ function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
   set_color normal
 end
 
+function __bobthefish_prompt_rubies -d 'Display current Ruby information'
+  switch "$theme_display_ruby"
+    case no; return
+    case auto
```

Comment:
Ok. I see. I'll fix that, not sure how to do it better. Just I'm going to traverse all parents of current directory and check how fast is it.